lib/renderer.js

```diff
@@ -33,7 +34,7 @@ exports.__esModule = {
   valid: true
 }
 
-const generatedTheme = JSON.parse(process.env.LANDR_THEME)
+const landrTheme = process.env.LANDR_THEME ? JSON.parse(process.env.LANDR_THEME) : defaultTheme
```

Comment:
This fixes the failing storybook. The generated theme is undefined when using storybook - falling back to the default one.